thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/SummaryResource.java

```diff
@@ -180,4 +181,69 @@ public String buildSummaryManualDimensionOrder(@QueryParam("dataset") String dat
     }
     return OBJECT_MAPPER.writeValueAsString(response);
   }
+
+  @GET
+  @Path(value = "/summary/autoRatioDimensionOrder")
```

Comment:
how about sharing the same endpoint with additive metrics and run a separate code path in the back end? Instead of making different frontend calls.

Reply:
The integration is difficult to do for now because a ratio metric needs to be broken down to `numerator` and `denominator`. Although the information is available, it cannot be used directly. 

Specifically, `ratio = id123/id456` where `id123` and `id456` are the numerator and denominator. We need to create another query to do a reverse lookup and get the original metric name. Otherwise, ThirdEye won't be able to get metrics by the `id` names.

I suggest that we do the integration in another PR. This PR will focus on the implementation of the Ratio Cube. Users can use it through API for now and start to give us feedback. Does that work for you?

Reply:
sure. sounds good.